app/scripts/models/reliers/oauth.js

```diff
@@ -159,7 +159,13 @@ define([
           //jshint camelcase: false
           // server version always takes precedent over the search parameter
           self.set('redirectUri', serviceInfo.redirect_uri);
+          self.set('termsUri', serviceInfo.terms_uri);
+          self.set('privacyUri', serviceInfo.privacy_uri);
+          self.set('trusted', serviceInfo.trusted);
```

Comment:
So yeah, this just happened on IRC:
```
08:55 rfkelly: nate, rfeeley: just want to sanity-check expectations on ToS and privacy links
08:55 rfkelly: on our next deployment we're expecting to ship the ability to display relier ToS and privacy links during the oauth flow
08:56 rfkelly: i.e. some "to continue you must agree to pocket's terms and privacy policy" text
08:56 clarkbw: rfkelly: the pocket landing page is going to show the ToS
08:57 nate: (which is already live now if you go through the sign-up process)
08:57 rfkelly: because that's the preferred place for it, or because we don't currently have the ability to show it during the flow?
08:59 nate: rfkelly: Preferred. My understanding from Mozilla legal was that they felt that the TOS needed to be on a Pocket branded page (not a FF one) to avoid confusion. Additionally, it let's us have a bit better control on how to communicate it.
```

Which sadly makes the terms/privacy fields we just added rather useless.  C'est la vie; I guess they'll come in handy for other third-party reliers in the future...